packages/toolpad-app/src/toolpad/Home.tsx

```diff
@@ -334,55 +511,119 @@ export default function Home() {
 
   const [deletedApp, setDeletedApp] = React.useState<null | App>(null);
 
+  const [tabIndex, setTabIndex] = React.useState(0);
+
+  const handleTabChange = React.useCallback((event: React.SyntheticEvent, newIndex: number) => {
+    setTabIndex(newIndex);
+  }, []);
+
+  const [viewMode, setViewMode] = React.useState<string>('list');
+
+  const handleViewModeChange = React.useCallback((event: React.MouseEvent, value: string) => {
+    setViewMode(value);
+  }, []);
+
   return (
-    <ToolpadShell>
+    <ToolpadShell navigation={<TabList value={tabIndex} handleTabChange={handleTabChange} />}>
       <AppDeleteDialog app={deletedApp} onClose={() => setDeletedApp(null)} />
-      <Container>
-        <Typography variant="h2">Apps</Typography>
-        <CreateAppDialog open={createDialogOpen} onClose={() => setCreateDialogOpen(false)} />
-
-        <Toolbar disableGutters>
-          <Button onClick={() => setCreateDialogOpen(true)}>Create New</Button>
-        </Toolbar>
-
-        <Box
-          sx={{
-            display: 'grid',
-            gridTemplateColumns: {
-              lg: 'repeat(4, 1fr)',
-              md: 'repeat(3, 1fr)',
-              sm: 'repeat(2, fr)',
-              xs: 'repeat(1, fr)',
-            },
-            gap: 2,
-          }}
-        >
-          {(() => {
-            switch (status) {
-              case 'loading':
-                return <AppCard />;
-              case 'error':
-                return <Alert severity="error">{(error as Error)?.message}</Alert>;
-              case 'success':
-                return apps.length > 0
-                  ? apps.map((app) => {
-                      const activeDeployment = activeDeploymentsByApp?.[app.id];
-                      return (
-                        <AppCard
-                          key={app.id}
-                          app={app}
-                          activeDeployment={activeDeployment}
-                          onDelete={() => setDeletedApp(app)}
-                        />
-                      );
-                    })
-                  : 'No apps yet';
-              default:
-                return '';
-            }
-          })()}
-        </Box>
-      </Container>
+      <TabPanel index={0} value={tabIndex}>
```

Comment:
Does this have to be part of this PR?

Reply:
This is to move the "Apps" heading into an "Apps" tab

Reply:
Is there a need for this right now? It seems a bit redundant. Also, it looks (and works) completely different from how this menu is in the editor. Let's leave it out of the scope of this PR.